JitsiConferenceEvents.js

```diff
@@ -57,6 +57,11 @@ export const CONNECTION_INTERRUPTED = 'conference.connectionInterrupted';
 export const CONNECTION_RESTORED = 'conference.connectionRestored';
 
 /**
+ * A connection to the video bridge's data channel has been established.
+ */
+export const DATA_CHANNEL_OPEN = 'conference.dataChannelOpen';
```

Comment:
That would make it inconsistent with RTCEvents.DATA_CHANNEL_OPEN. I'm not opposed to changing it, though.